cgi/org.pl

```diff
@@ -384,6 +386,30 @@
 		}
 	}
 
+	elsif ($type eq 'admin_status') {
+		if (is_user_in_org_group($org_ref, $User_id, "admins") or $admin or $User{pro_moderator}) {
+			foreach my $user_id (sort keys %{$org_ref->{members}}) {
+				my $checkbox_name = "admin_status_hidden_" . $user_id;
+				my $is_admin = single_param($checkbox_name) ? 1 : 0;
+				$log->debug("checkbox name", {checkss => $checkbox_name, statuss => $is_admin}) if $log->is_debug();
+
+				if ($is_admin) {
+					add_user_to_org($org_ref, $user_id, ["admins"]);
+					unless (grep {$_ == $user_id} @admin_status) {
+						push @admin_status, $user_id;
+
+					}
+				}
+				else {
+					remove_user_from_org($org_ref, $user_id, ["admins"]);
+					@admin_status = grep {$_ != $user_id} @admin_status;
+				}
+				store_org($org_ref);
+			}
```

Comment:
again it's quite ok, but I would rather build a list of admins according to the form sent, and then compare it to the current list of the org and only act on differences.
(if you sort them before it's more easy to compare lists).

BTW @stephanegigandet could be useful to have libs like [Array::Diff](https://metacpan.org/pod/Array::Diff) for this kind of operations.